src/PlaneWaveBasis.jl

```diff
@@ -112,8 +112,17 @@ Base.Broadcast.broadcastable(basis::PlaneWaveBasis) = Ref(basis)
 
 Base.eltype(::PlaneWaveBasis{T}) where {T} = T
 
+function select_G_vector(model, G, k, Ecut)
+    # Strict inequality for ModifiedKinetic
+    if (model.model_name == "ModifiedKinetic")
+        return  (sum(abs2, model.recip_lattice*(G+k))/2 < Ecut)
+    end
+    (sum(abs2, model.recip_lattice*(G+k))/2 ≤ Ecut)
+end
```

Comment:
that's a hack... just have your blow up function return a very large value at Ecut if you're really concerned about it? (see eg floatmax)